miso-web/src/main/webapp/scripts/list_partition.js

```diff
@@ -277,10 +275,14 @@ ListTarget.partition = {
       "bSortDirection": true
     }, {
       "sTitle": "Pool",
-      "mData": "pool",
+      "mData": function(full) {
+        return full.pool ? full.pool.name + ' (' + full.pool.alias + ')' : '(none)';
+      },
       "include": config.showPool,
       "iSortPriority": 0,
-      "mRender": WarningTarget.partition.tableWarnings
+      "mRender": Warning.tableWarningRenderer(WarningTarget.partition, function(full) {
+        return '/miso/pool/' + full.pool.id;
```

Comment:
`Cannot read property 'id' of null` (when there's no pool in the lane)

Reply:
fixed